src/ir/auth_logic/souffle_emitter_test.cc

```diff
@@ -64,7 +64,6 @@ TEST(EmitterTestSuite, EmptyAuthLogicTest) {
 .type Number <: symbol
 .decl grounded_dummy(dummy_param : DummyType)
 .decl says_canActAs(speaker : Principal, p1 : Principal, p2 : Principal)
-.decl says_isNumber(speaker : Principal, x : Number)
```

Comment:
Why is this not there anymore?

Reply:
this is generated by the universe relation compiler pass now.